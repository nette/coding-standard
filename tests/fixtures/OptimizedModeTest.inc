<?php // {"optimizedFunctionsOnly": true}

namespace App;

use function trim;
use function count;

class MyClass
{
    public function run()
    {
        // Optimized function, should be managed.
        $len = \strlen('foo');

        // Not an optimized function, should be ignored by the sniff.
        trim(' bar ');
    }
}
