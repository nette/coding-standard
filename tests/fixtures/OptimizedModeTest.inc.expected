<?php // {"optimizedFunctionsOnly": true}

namespace App;

use function strlen, trim;

class MyClass
{
    public function run()
    {
        // Optimized function, should be managed.
        $len = strlen('foo');

        // Not an optimized function, should be ignored by the sniff.
        trim(' bar ');
    }
}
